keel-core/src/main/kotlin/com/netflix/spinnaker/keel/services/ApplicationService.kt

```diff
@@ -91,9 +91,8 @@ class ApplicationService(
   private val RESOURCE_SUMMARY_CONSTRUCT_DURATION_ID = "keel.api.resource.summary.duration"
   private val ENV_SUMMARY_CONSTRUCT_DURATION_ID = "keel.api.environment.summary.duration"
   private val ARTIFACT_SUMMARY_CONSTRUCT_DURATION_ID = "keel.api.artifact.summary.duration"
-  private val CONSTRAINT_SUMMARY_CONSTRUCT_DURATION_ID = "keel.api.constraint.summary.duration"
+  private val ARTIFACT_IN_ENV_SUMMARY_CONSTRUCT_DURATION = "keel.api.artifact.in.environment.summary.duration"
```

Comment:
Will you be able to compare this new metric with the other two that it's replacing? Just trying to understand if you'll be able to do a before/after comparison given that you're changing the metrics.

Reply:
Good question. The metric I removed was useless to measure (it was taking 25millis each time, and was measuring constructing things for each version). So I removed it. I added another metric now that I better understand how the function works. The main metric I'm measuring (`keel.api.artifact.summary.duration`, with an approximate average of 6 seconds) will remain the same so that it's useful.